packages/jsx/jsx-runtime.js

```diff
@@ -0,0 +1,23 @@
+import { h, text } from "hyperapp"
+
+const childNode = (child) => typeof child === "string" || typeof child === "number" ? text(child) : child;
```

Comment:
Just my 2c
I use similar condition in my private project and have to check on `typeof child === "bigint"` too.

Reply:
Wow, true. What about `boolean` also? What about dates?

What would you expect this render:
```js
<ul>
  <li>{true}</li>
  <li>{false}</li>
  <li>{Date.now()}</li>
</li>
```

Just playing 😈's advocate, really.

Reply:
Wow, true. What about `boolean` also? What about dates?

What would you expect this to render?
```js
<ul>
  <li>{true}</li>
  <li>{false}</li>
  <li>{Date.now()}</li>
</li>
```

Just playing 😈's advocate here.

Reply:
Wow, true! And what about `boolean` types? What about dates? Anything else? 🤔 

What would you expect this to render?

```js
<ul>
  <li>{true}</li>
  <li>{false}</li>
  <li>{(new Date)}</li>
</li>
```

Just playing 😈's advocate here.

Reply:
It's sensible reasoning. Maybe a better approach would be:

```js
child => ('node' in child) ? child : text(child.toString())
```

Reply:
@zaceno Hmm not sure why but I was having some issues with the above function. I ended with this instead:

```js
const textNodes = ["string", "number", "bigint"];
const childNode = (child) => textNodes.includes(typeof child) ? text(child) : child;
```

Reply:
I do not think we can include `false` aka `boolean` as it would prevent usage of `<div>{condition && element}</div>`.
Not sure about `Date`. Anyway, `typeof new Date` is "object". There had to be added `instanceof` base check.

Reply:
@mshgh @zaceno @jorgebucaran Sorry for the late reply, here's new solution that seems to cover to everything:

```js
const isVNode = (child) => typeof child === "object" && ('node' in child)

const childNode = (child) => {
  if (child) {
    return isVNode(child) ? child : text(child)
  }
}
```

I updated the PR as well, let me know what you think!